desktop/forge.config.ts

```diff
@@ -1,23 +1,23 @@
-import type {ForgeConfig} from '@electron-forge/shared-types'
-import {MakerDMG} from '@electron-forge/maker-dmg'
-import {WebpackPlugin} from '@electron-forge/plugin-webpack'
-import {MakerZIP} from '@electron-forge/maker-zip'
-const ForgeExternalsPlugin = require('@timfish/forge-externals-plugin')
+import type { ForgeConfig } from "@electron-forge/shared-types";
+import { MakerDMG } from "@electron-forge/maker-dmg";
+import { WebpackPlugin } from "@electron-forge/plugin-webpack";
+import { MakerZIP } from "@electron-forge/maker-zip";
+const ForgeExternalsPlugin = require("@timfish/forge-externals-plugin");
 
 // import {CopyPlugin} from 'copy-webpack-plugin'
 
-import {mainConfig} from './webpack.main.config'
-import {rendererConfig} from './webpack.renderer.config'
-import dotenv from 'dotenv'
-dotenv.config({path: '../../.env'})
+import { mainConfig } from "./webpack.main.config";
+import { rendererConfig } from "./webpack.renderer.config";
+import dotenv from "dotenv";
+dotenv.config({ path: "../../.env" });
 
-const fs = require('fs-extra')
-const path = require('path')
-const {spawn} = require('child_process')
+const fs = require("fs-extra");
+const path = require("path");
+const { spawn } = require("child_process");
 
 const config: ForgeConfig = {
   packagerConfig: {
-    asar: true,
+    asar: false,
```

Comment:
Hi @RadientBrain, apologies, I should have specified. I would like to keep `asar` enabled because it seems to have a few benefits: https://www.electronjs.org/docs/latest/tutorial/asar-archives

Did you manage to get it working without turning asar off?

Reply:
Though actually, either way, I did test out `asar: false` but I'm still seeing the same error.  Confirming this works for you with `npm run make` and not only `npm run dev`?